Source/WebInspectorUI/UserInterface/Base/BrowserInspectorFrontendHost.js

```diff
@@ -233,18 +228,54 @@ if (!window.InspectorFrontendHost) {
             return false;
         }
 
-        save(url, content, base64encoded, forceSaveAs)
+        save(saveDatas, forceSaveAs)
```

Comment:
Result is the same either way, but might be worth updating the signature of `canSave()` to `canSave(saveMode)` for completeness

Reply:
Right, I missed that one. I will update it before landing.